src/engine/enginebuffer.cpp

```diff
@@ -185,6 +185,8 @@ EngineBuffer::EngineBuffer(QString group, UserSettingsPointer pConfig,
             this, SLOT(slotEjectTrack(double)),
             Qt::DirectConnection);
 
+    m_pTrackLoaded = new ControlPushButton(ConfigKey(m_group, "track_loaded"));
```

Comment:
I think this is not a PushButton just
new ControlObject() fits better   

Since this is a read only type, you should use: 
connectValueChangeRequest
and setAndConfirm 
like here: 
https://github.com/mixxxdj/mixxx/blob/5fe4c3f378d74ab45698cc9cd9e8d8cbe5f3333f/src/engine/sidechain/enginebroadcast.cpp#L59